routers/web/repo/pull.go

```diff
@@ -1411,3 +1412,32 @@ func UpdatePullRequestTarget(ctx *context.Context) {
 		"base_branch": pr.BaseBranch,
 	})
 }
+
+// SetAllowEdits allow edits from maintainers to PRs
+func SetAllowEdits(ctx *context.Context) {
```

Comment:
this function is better in pull_services - as it will be needed if sething this potion via api is added